### PR TITLE
Deposit collateral event

### PR DESCRIPTION
**Pull request by ororopickpocket**

### CODE REVIEW

File: contracts/events/LoanMaintenanceEvents.sol [hunk 1]
```diff
@@ -0,0 +1,10 @@
+pragma solidity 0.5.17;
+
+contract LoanMaintenanceEvents {
+    
+    event DepositCollateral(
```
Comment (kurt-wagner) [inline contracts/events/LoanMaintenanceEvents.sol:5]: Would not be a good idea to add the sender?

File: tests/protocol/addingMargin/test_deposit_collareral_using_TestToken.py [hunk 2]
```diff
@@ -17,14 +17,20 @@ def test_deposit_collateral(sovryn,set_demand_curve,lend_to_pool,open_margin_tra
...
+    deposit_amount = startCollateral/2
     
     #deposit collateral to add margin to the loan created above
-    RBTC.approve(sovryn, startCollateral/2)
-    sovryn.depositCollateral(loan_id, startCollateral/2)
+    RBTC.approve(sovryn, deposit_amount)
+    tx = sovryn.depositCollateral(loan_id, deposit_amount)
+    
+    #verify the deposit collateral event
+    print(tx.info())
```
Comment (kurt-wagner) [inline tests/protocol/addingMargin/test_deposit_collareral_using_TestToken.py:27]: You don't need to use `print`. It also works just with `tx.info()`

**Review by kurt-wagner — APPROVED** [review]

File: contracts/events/LoanMaintenanceEvents.sol (hunk 1 above)
Comment (ororopickpocket) [reply contracts/events/LoanMaintenanceEvents.sol:5]: Victor just asked for loanId and amount, but yes, maybe it'd turnout helpful at some point in the future to have the sender as well.

File: tests/protocol/addingMargin/test_deposit_collareral_using_TestToken.py (hunk 2 above)
Comment (ororopickpocket) [reply tests/protocol/addingMargin/test_deposit_collareral_using_TestToken.py:27]: thanks. if you've got more hints, go ahead ;)

**Comment by ororopickpocket** [comment]
The toggle function pause is now not working anymore. that's because it's now handled differently on the settings contract, but the logic contract wasn't updated (checkPause is not working anymore). Could you also merge the changes of the loan token logic contract?

**Pull request merged**